packages/app/src/app/components/CreateNewSandbox/CreateSandbox/CreateSandbox.tsx

```diff
@@ -211,7 +227,12 @@ export const CreateSandbox: React.FC<CreateSandboxProps> = ({
           <ModalSidebar>
             {viewState === 'initial' ? (
               <Stack direction="vertical">
-                <Tabs {...tabState} aria-label="Create new">
+                <Tabs
+                  {...tabState}
+                  // TODO: Any click on the tabs will reset the search query
+                  onClick={() => setSearchQuery('')}
```

Comment:
This is still a todo because this should be part of clicking the individual tabs instead of clicking the list? Right now you can click the empty space between "Import" and "Team" tabs and it clears the search query.

Reply:
Right, I just pushed a fix for this moving the reset in a useEffect to be triggered by an explicit tab change